src/client/serialize/daos_serialize.c

```diff
@@ -848,6 +1011,26 @@ deserialize_props(daos_handle_t poh, hid_t file_id, daos_prop_t **_prop,
 		}
 		prop_num++;
 	}
+    if (H5Aexists(file_id, "DAOS_PROP_CO_ROOTS") > 0) {
+        type = DAOS_PROP_CO_ROOTS;
+        prop->dpp_entries[prop_num].dpe_type = type;
+        entry = &prop->dpp_entries[prop_num];
+        rc = deserialize_roots(file_id, entry, "DAOS_PROP_CO_ROOTS");
+        if (rc != 0) {
+            D_GOTO(out, rc);
+        }
+        prop_num++;
+    }
+    if (H5Aexists(file_id, "DAOS_PROP_CO_EC_CELL_SZ") > 0) {
+        type = DAOS_PROP_CO_EC_CELL_SZ;
+        prop->dpp_entries[prop_num].dpe_type = type;
+        entry = &prop->dpp_entries[prop_num];
+        rc = deserialize_uint(file_id, &entry->dpe_val, "DAOS_PROP_CO_EC_CELL_SZ");
+        if (rc != 0) {
```

Comment:
(style)  code indent should use tabs where possible
(style)  suspect code indent for conditional statements (8, 12)